prime-router/build.gradle.kts

```diff
@@ -713,7 +713,7 @@ dependencies {
     implementation("commons-io:commons-io:2.11.0")
     implementation("org.postgresql:postgresql:42.3.3")
     implementation("com.zaxxer:HikariCP:5.0.1")
-    implementation("org.flywaydb:flyway-core:8.5.5")
+    implementation("org.flywaydb:flyway-core:8.5.8")
```

Comment:
I added this bump per Chris Winters' excellent suggestion.  Worked locally and remotely.